app/controllers/searches_controller.rb

```diff
@@ -5,6 +5,10 @@ def show
     return unless params[:query]&.is_a?(String)
     @error_msg, @gems = ElasticSearcher.new(params[:query], page: @page).search
     limit_total_count if @gems.total_count > Gemcutter::SEARCH_MAX_PAGES * Rubygem.default_per_page
+
+    exact_match = Rubygem.name_is(params[:query]).first
+    @yanked_gem = exact_match unless exact_match&.indexed_versions?
+    @yanked_filter = params[:yanked]
```

Comment:
maybe check if params is  `true` and set var to boolean?

Reply:
we want to make sure we aren't using any user input data directly in view. they can send all kinds of garbage.

Reply:
Done! :+1: